app/models/dns.rb

```diff
@@ -0,0 +1,11 @@
+class Dns < Parameter
+  Dns.reset_callbacks(:validate)
```

Comment:
Could you leave a comment explaining why this is needed? I know the reason, but at first glance it'll be difficult to know, especially for new people reading this code.